src/hooks/useSwapAdjustedAmounts.ts

```diff
@@ -40,7 +47,10 @@ export default function useSwapAdjustedAmounts(tradeDetails: Quote) {
     (tradeDetails.sellTokenAddress === ETH_ADDRESS && tradeDetails.buyTokenAddress === WETH_ARBITRUM_ADDRESS) ||
     // zora eth <-> weth swap
     (tradeDetails.buyTokenAddress === ETH_ADDRESS && tradeDetails.sellTokenAddress === WETH_ZORA_ADDRESS) ||
-    (tradeDetails.sellTokenAddress === ETH_ADDRESS && tradeDetails.buyTokenAddress === WETH_ZORA_ADDRESS)
+    (tradeDetails.sellTokenAddress === ETH_ADDRESS && tradeDetails.buyTokenAddress === WETH_ZORA_ADDRESS) ||
+    // degen <-> wdegen swap
+    (tradeDetails.buyTokenAddress === ETH_ADDRESS && tradeDetails.sellTokenAddress === WDEGEN_DEGEN_CHAIN_ADDRESS) ||
+    (tradeDetails.sellTokenAddress === ETH_ADDRESS && tradeDetails.buyTokenAddress === WDEGEN_DEGEN_CHAIN_ADDRESS)
```

Comment:
I remember having to touch this when we were having some swaps issues when wrapping eth to weth on zora. We need to explore a way to no longer have to add entries here.